### ISSUE TITLE
Watch folder

### ISSUE THREAD

**Issue by pyalex2** [LycheeOrg/Lychee#242]
Dear, 
is there some planning to add a watching routine to import on backend new photos added in a specific folder?

Have a great day.

**Comment by d7415** [comment]
Yes. It may be a while, but it is planned.

**Comment by ildyria** [comment]
@d7415 should we move this to the V4 ? :)

**Comment by d7415** [comment]
> @d7415 should we move this to the V4 ? :)

Yeah. I never finished it, and it makes more sense to put it on the more active project.
